src/format/scan.rs

```diff
@@ -198,6 +198,27 @@ pub(super) fn space(s: &str) -> ParseResult<&str> {
     }
 }
 
+/// Returns slice remaining after first char.
+/// If <=1 chars in `s` then return an empty slice
+pub(super) fn s_next(s: &str) -> &str {
+    match s.char_indices().nth(1) {
+        Some((offset, _)) => &s[offset..],
+        None => {
+            // one or zero chars in `s`, return empty string
+            &s[s.len()..]
+        }
+    }
+}
+
+/// If the first `char` is whitespace then consume it and return `s`.
+/// Else return `s`.
+pub(super) fn space1(s: &str) -> &str {
```

Comment:
potentially this could be better named `trim1` - to contrast from the existing `space` function which is fallible and asserts the existence of at least one space

Reply:
I'll rename this.